.github/actions/run-gradle/action.yml

```diff
@@ -16,8 +16,8 @@
 # under the License.
 #
 ---
-name: "Gradle Setup"
-description: "Setup Java and Gradle"
+name: "Run Gradle Tests"
```

Comment:
Could you please use "Run Tests with Gradle"? It makes more sense to me.

Reply:
It makes more sense. I’ve made the change accordingly.